### PR TITLE
Next release

### PR DESCRIPTION
**Pull request by bengourley**
Fixes for a bunch of bugs that accrued while we were working on the universal JS release, plus the Nuxt example which didn't quite make it in before the v5.0.0 release.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,18 @@
...
+### Fixed
+
+- Ensure objects with a `null` prototype or bad `toString()` implementation don't cause an error in console breadcrumbs (#429)
+- Ensure user ip is not collected when `collectUserIp=false` but user.id is explicitly `undefined` (#428)
+- Ensure previous `window.onerror` callback is always called (#427)
+- Ensure previous `window.onreadystatechange` callback is called (#426)
+
+### Added
+
+- Nuxt.js example (#425)
```
Comment (kattrali) [inline CHANGELOG.md]: Does this involve some source changes to the library?

File: packages/plugin-console-breadcrumbs/test/console-breadcrumbs.test.js [hunk 2]
```diff
@@ -31,4 +31,13 @@ describe('plugin: console breadcrumbs', () => {
     // undo the global side effects of wrapping console.* for the rest of the tests
     plugin.destroy()
   })
+
+  it('should not throw when an object without toString is logged', () => {
+    const c = new Client(VALID_NOTIFIER)
+    c.setOptions({ apiKey: 'aaaa-aaaa-aaaa-aaaa' })
+    c.configure()
+    c.use(plugin)
+    expect(() => console.log(Object.create(null))).not.toThrow()
```
Comment (kattrali) [inline packages/plugin-console-breadcrumbs/test/console-breadcrumbs.test.js]: There should also be an assertion about what effect this has on breadcrumbs. Did it create one?

**Comment by kattrali** [comment]
Looks good to me, added a couple of questions inline.

File: CHANGELOG.md (hunk 1 above)
Comment (bengourley) [reply CHANGELOG.md]: Nope, just a new directory in `examples`

**Pull request merged**